### DISCUSSION TITLE
time-dependent PidControl

### DISCUSSION THREAD

**Discussion by harm-nomden-sweco** [Deltares/Ribasim#745] [General]
We were thinking about the implementation of the PidControl. We know it is possible to use time-dependent target levels. Is it possible to deactivate the PidControl during certain periods? like only controlling water levels during summer.

**Comment by visr** [comment]
`PiDControl / static` has an `active` column, but `PiDControl / time` does not. If I remember correctly that wasn't added directly since it was harder to do right. Have you tried setting the proportional, integral and derivative parameters all to zero? Possibly that has the same effect.
